java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -132,18 +133,12 @@ public void visitFile(@Nullable Tree parsedTree) {
   }
 
   protected JavaFileScannerContext createScannerContext(
-    CompilationUnitTree tree, SemanticModel semanticModel, SonarComponents sonarComponents, boolean fileParsed) {
-    return new DefaultJavaFileScannerContext(
-      tree,
-      currentFile,
-      semanticModel,
-      sonarComponents,
-      javaVersion,
-      fileParsed);
+    CompilationUnitTree tree, SemanticModel semanticModel, SonarComponents sonarComponents, File currentFile, boolean fileParsed) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "sonarComponents". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)